onebusaway-android/src/main/java/org/onebusaway/android/report/ui/ReportTypeListFragment.java

```diff
@@ -129,9 +129,12 @@ public void onItemClick(AdapterView<?> adapterView, View view, int i, long l) {
                     .getStringExtra(BaseReportActivity.LOCATION_STRING);
             UIUtils.sendEmail(getActivity(), email, locationString);
 
+            String gaEventLabel = locationString != null ?
+                    getString(R.string.analytics_label_app_feedback) :
+                    getString(R.string.analytics_label_app_feedback_without_location);
```

Comment:
@barbeau If there is no location, we send `Send Email Link without location` as a GA label.